server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -173,7 +172,8 @@ func (e *EventChannel) MarkInvocationDisconnected(ctx context.Context, iid strin
 		return err
 	}
 	if e.logWriter != nil {
-		invocation.LastChunkId = chunkstore.ChunkIndexAsStringId(e.logWriter.GetLastChunkIndex())
+		e.logWriter.Close()
+		invocation.LastChunkId = chunkstore.ChunkIndexAsStringId(e.logWriter.ChunkstoreWriter.GetLastChunkIndex())
```

Comment:
nit: rather than reaching into a struct (like logWriter) for publicly exposed members -- could you have a GetLastChunkIndex() method on logWriter that internally calls the chunkstore writer?

Reply:
Sure, I can do that.